packages/ses-ava/src/ses-ava-test.js

```diff
@@ -75,6 +75,16 @@ const overrideList = [
   'only',
 ];
 
+// Helper to explicitely harden an object including its prototype chain
```

Comment:
Should the changes in this file be moved into #1914 ? They don't really make sense in this PR.

Reply:
Yeah I debated this. Happy to move to #1914